<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: 	rgb(245,255,250);
background-color: 	rgb(238,238,228);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="create_username">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Calibri (Body)&quot;;
background-color: rgb(21,76,121);
color: rgb(255, 255, 255);
border-radius:5px;



</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="create_password">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Calibri (Body)&quot;;
background-color: rgb(21,76,121);
color: rgb(255, 255, 255);
border-radius:5px;



</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Calibri (Body)&quot;;
color:   rgb(21,76,121);
 </string>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Calibri (Body)&quot;;
background-color: rgb(60,179,113);
color: rgb(255, 255, 255);
border-radius:10px;



</string>
    </property>
    <property name="text">
     <string>Create User</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Calibri (Body)&quot;;
color:   rgb(21,76,121);
 </string>
    </property>
    <property name="text">
     <string>Username :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confir_password">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Calibri (Body)&quot;;
background-color: rgb(21,76,121);
color: rgb(255, 255, 255);
border-radius:5px;



</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Calibri (Body)&quot;;
color:   rgb(21,76,121);
 </string>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>162</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Calibri (Body)&quot;;
color:   rgb(21,76,121);
 </string>
    </property>
    <property name="text">
     <string>confirm Password :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>212</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Calibri (Body)&quot;;
color:   rgb(21,76,121);
 </string>
    </property>
    <property name="text">
     <string>Gmail :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="create_gmail">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Calibri (Body)&quot;;
background-color: rgb(21,76,121);
color: rgb(255, 255, 255);
border-radius:5px;



</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Gmail</string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>create_username</zorder>
   <zorder>create_password</zorder>
   <zorder>label</zorder>
   <zorder>CreateButton</zorder>
   <zorder>label_2</zorder>
   <zorder>confir_password</zorder>
   <zorder>label_3</zorder>
   <zorder>label_5</zorder>
   <zorder>create_gmail</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
